### ISSUE TITLE
Crystal 0.27.0 update to Tempfile

### ISSUE THREAD

**Issue by Phiction** [amberframework/amber#976]
### Description

i'm running into an issue when running `amber watch`.

```bash
amber watch
01:20:33 Watcher    | (INFO) Watching 18 files (server reload)...
01:20:33 Watcher    | (INFO) Building project Amber...
Error in src/amber.cr:1: while requiring "../config/*"

require "../config/*"
^

in config/application.cr:3: while requiring "amber"

require "amber"
^

in lib/amber/src/amber.cr:16: while requiring "./amber/router/context"

require "./amber/router/context"
^

in lib/amber/src/amber/router/context.cr:1: while requiring "./request"

require "./request"
^

in lib/amber/src/amber/router/request.cr:1: while requiring "./params"

require "./params"
^

in lib/amber/src/amber/router/params.cr:3: while requiring "./file"

require "./file"
^

in lib/amber/src/amber/router/file.cr:2: while requiring "tempfile": can't find file 'tempfile'

If you're trying to require a shard:
- Did you remember to run `shards install`?
- Did you make sure you're running the compiler in the same directory as your shard.yml?

require "tempfile"
^

01:20:34 Watcher    | (INFO) Compile time errors detected. Shutting down...
```

i'm guessing this has to do with crystals new update here?

https://crystal-lang.org/2018/11/01/crystal-0.27.0-released.html
https://github.com/crystal-lang/crystal/pull/6485

> Working with temporal files and directories used to require using Tempfile class. Now the creation of these kind of files are handled by File.tempfile or File.tempname. This change also tidy up the usage of prefix, suffix and default temp path.

### Steps to Reproduce

1. follow quick start guide https://docs.amberframework.org/amber/getting-started/quick-start

**Expected behavior:** amber watches
**Actual behavior:** amber shuts down
**Reproduces how often:**100%

### Versions

crystal
```bash
Crystal 0.27.0 (2018-11-04)

LLVM: 6.0.1
Default target: x86_64-apple-macosx
```

amber
```bash
Amber CLI (amberframework.org) - v0.10.0
```

### Additional Information

danke

**Comment by Phiction** [comment]
i saw there was a closed issue but it had no replies.

**Comment by elorest** [comment]
This will be fixed as soon as we release Amber 0.11 later today. Use Crystal 0.26.1 with Amber 0.10

**Comment by Phiction** [comment]
saw the release. thank you
